src/main/play-26/twirl/uk/gov/hmrc/govukfrontend/views/components/govukInput.scala.html

```diff
@@ -65,7 +65,13 @@
 
 @inputSnippet(describedBy: String) = {
   @defining {
-    id="@id" name="@name" type="@inputType"@value.mapNonEmpty { value => value="@value" }@if(describedBy.nonEmpty) { aria-describedby="@describedBy"}
+    id="@id"
+    name="@name"
+    type="@inputType"
+    @if(spellcheck.nonEmpty) {
+      spellcheck="@spellcheck"
+    }
+    @value.mapNonEmpty {value => value="@value" }@if(describedBy.nonEmpty) { aria-describedby="@describedBy"}
```

Comment:
I notice the formatting here has changed, although the code itself looks good - is this change deliberate?